internal/controllers/inventory_controller.go

```diff
@@ -1055,7 +1066,7 @@ func (t *reconcilerTask) deployServer(ctx context.Context, serverName string) (u
 					{
 						Name:            utils.ServerContainerName,
 						Image:           image,
-						ImagePullPolicy: corev1.PullAlways,
+						ImagePullPolicy: corev1.PullIfNotPresent,
```

Comment:
This is usually a recommendation from OCP to use PullIfNotPresent. Especially important for SNO context where resource are constrained. Cant find the docs though where it was mentioned...but this is the "correct" way 🤔

Reply:
It might be. However, I was wondering why this change on this pr.

These are for the servers, right? It won't run in the SNO. Just a concern because we are in the developing phase. I usually modify the quay image (same name) and recreate the operator to pick up latest changes.

Reply:
@mlguerrero12  If your version tag is set to `latest` it will automatically pull the latest even if it is set to `IfNotPresent`.

Reply:
@alegacy that's not k8s works https://kubernetes.io/docs/concepts/containers/images/#required-image-pull. Am I missing something in our code?